<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="about">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>431</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset resource="photo.qrc">
       <normaloff>:/icons/file.png</normaloff>:/icons/file.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Normal Files</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="placeholderText">
       <string>URL Here</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Path to save file</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>381</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="photo.qrc">
        <normaloff>:/icons/br.png</normaloff>:/icons/br.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">  background-color: DodgerBlue;
  border: none;
  color: white;
  padding: 12px 30px;
  font-size: 20px</string>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset resource="photo.qrc">
       <normaloff>:/icons/youtube.png</normaloff>:/icons/youtube.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Youtube</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>391</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset resource="photo.qrc">
         <normaloff>:/icons/one.png</normaloff>:/icons/one.png</iconset>
       </attribute>
       <attribute name="title">
        <string>One video</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Video URL Here</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>381</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Path to save video</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>190</y>
          <width>201</width>
          <height>51</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: DodgerBlue;
  border: none;
  color: white;
  padding: 12px 30px;
  font-size: 20px</string>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Video Quality</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>150</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>20</y>
          <width>31</width>
          <height>28</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click To fetch video Info from youtube&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/pull.png</normaloff>:/icons/pull.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>60</y>
          <width>31</width>
          <height>28</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/br.png</normaloff>:/icons/br.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="icon">
        <iconset resource="photo.qrc">
         <normaloff>:/icons/multi.png</normaloff>:/icons/multi.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Playlist</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Path to save video</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>381</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Video URL Here</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>60</y>
          <width>31</width>
          <height>28</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="photo.qrc">
          <normaloff>:/icons/br.png</normaloff>:/icons/br.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>160</y>
          <width>381</width>
          <height>31</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>230</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Of</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>220</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>220</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>280</y>
          <width>201</width>
          <height>51</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">  background-color: DodgerBlue;
  border: none;
  color: white;
  padding: 12px 30px;
  font-size: 20px</string>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>27</x>
          <y>110</y>
          <width>541</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>160</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>0 KB/S</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>About Me</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>-70</y>
        <width>581</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>This is my First Desktop App with Paython and PQT5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>581</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Follow this link to get the source Code</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://github.com/ahmed3bead&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;View Code Source&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>281</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://www.youtube.com/watch?v=CGEu9C44xCc&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Click here to see the tutorial &lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>270</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;a href='https://www.facebook.com/A7med3Bead'&gt;&lt;img src=&quot;:/icons/facebook.png&quot; width=&quot;90&quot; /&gt;&lt;/a&gt;</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
